app/views/content/blog/abigails-career-progression-story.md

```diff
@@ -66,4 +66,4 @@ It's a time when you get lots of extra support to help establish yourself as a t
 
 _Induction used to last 1 year, and teachers would be called 'newly qualified teachers' (NQTs)._
 
-_If you’ve been inspired by Abigail’s story, find out more about [career progression opportunities and teacher pay scales](/salaries-and-benefits#career-progression)._
+If you’ve been inspired by Abigail’s story, find out more about [career progression opportunities and teacher pay scales](/salaries-and-benefits#career-progression).
```

Comment:
Would suggest that we include the two paras above in the footer ("For the first 2 years..." and "Induction used to last....") - whilst these aren't strictly signposting, they don't fit in with the content above as they are informational rather than case study

Reply:
This is tricky as the footer is styled by the stylesheet, it only targets the final paragraph. Perhaps we can reword?